examples/adc.rs

```diff
@@ -1,26 +1,23 @@
 #![deny(unsafe_code)]
+#![deny(warnings)]
```

Comment:
I discovered not long ago that this is an anti-pattern. See [here](https://github.com/rust-unofficial/patterns/blob/master/anti_patterns/deny-warnings.md)

Reply:
Then I will remove this from other examples.